app/styles/_overview.less

```diff
@@ -43,12 +43,11 @@
   margin-bottom: 5px;
   margin-top: 5px;
   padding: 5px;
-  .truncate();
   .mini-log-content {
     line-height: @mini-log-line-height;
     min-height: (@mini-log-num-lines * @mini-log-line-height);
     .mini-log-line {
-      white-space: pre;
```

Comment:
This improves our newline handling, but will also collapse spaces and tabs. Trying to decide if that's a problem.